actionview/lib/action_view/helpers/url_helper.rb

```diff
@@ -172,6 +172,39 @@ def _filtered_referrer # :nodoc:
       #   link_to "External link", "http://www.rubyonrails.org/", target: "_blank", rel: "nofollow"
       #   # => <a href="http://www.rubyonrails.org/" target="_blank" rel="nofollow">External link</a>
       #
+      # ==== Turbo
+      #
+      # Rails 7 ships with Turbo enabled by default. The <tt>data-turbo-method</tt> HTML attribute
+      # can be used to perform a Turbo link visit with a different HTTP method. {Consult the
+      # Turbo Handbook for more information on performing visits with different
+      # methods.}[https://turbo.hotwired.dev/handbook/drive#performing-visits-with-a-different-method]
+      #
+      # A common use case is to insert links to perform update or delete operations
+      # (i.e. non-<tt>GET</tt> requests) in a form.
+      #
+      # For example, you might use +button_to+ in an edit form to delete a record:
+      #
+      #   button_to "Delete profile", @profile, method: :delete
+      #
+      # Since forms cannot appear inside another form,
+      # +button_to+ will not work here as it returns a form.
+      #
+      # Instead, you should use +link_to+ with +data-turbo-method+
+      # to perform the +DELETE+ request:
+      #
+      #   link_to "Delete profile", @profile, data: { turbo_method: :delete }
+      #   # => <a href="/profiles/1" data-turbo-method="delete">Delete profile</a>
+      #
+      # Using <tt>data-turbo-method</tt> is not ideal for accessibility
+      # but it is useful in scenarios like the above.
+      #
+      # The <tt>data-turbo-confirm</tt> HTML attribute adds a confirmation dialog to the link:
+      #
+      #   link_to "Visit Other Site", "https://rubyonrails.org/", data: { turbo_confirm: "Are you sure?" }
+      #   # => <a href="https://rubyonrails.org/" data-turbo-confirm="Are you sure?">
+      #          Visit Other Site
+      #        </a>
```

Comment:
```suggestion
      #   # => <a href="https://rubyonrails.org/" data-turbo-confirm="Are you sure?">Visit Other Site</a>
```

No need to break up the lines here IMO